wolox-books/src/app/screens/list-books/list-books.component.ts

```diff
@@ -1,12 +1,29 @@
 import { Component, OnInit } from '@angular/core';
 
+import { ListBooksService } from 'src/app/services/list-books.service';
+
 @Component({
   selector: 'app-list-books',
   templateUrl: './list-books.component.html',
   styleUrls: ['./list-books.component.scss']
 })
-export class ListBooksComponent {
+export class ListBooksComponent implements OnInit {
+
+  books: any;
```

Comment:
Create a model for books, here's how you can do it https://thinkster.io/tutorials/typescript-models-angular-2